packages/store/src/types.js

```diff
@@ -6,103 +6,25 @@
 /** @template T @typedef {import('@endo/marshal').CopyRecord<T>} CopyRecord */
 /** @template T @typedef {import('@endo/marshal').CopyArray<T>} CopyArray */
 /** @typedef {import('@endo/marshal').Checker} Checker */
+/** @typedef {import('@endo/marshal/src/rankOrder').RankCompare} RankCompare */
 
 /**
  * @typedef {Passable} Key
- *
- * Keys are Passable arbitrarily-nested pass-by-copy containers
- * (CopyArray, CopyRecord, CopySet, CopyBag, CopyMap) in which every
- * non-container leaf is either a Passable primitive value or a Remotable (a
- * remotely-accessible object or presence for a remote object), or such leaves
- * in isolation with no container.
- *
- * Keys are so named because they can be used as keys in CopyMaps and
- * [agoric-sdk Stores](https://github.com/Agoric/agoric-sdk/blob/master/packages/store/docs/store-taxonomy.md),
- * and as elements in CopySets and CopyBags.
- *
- * Keys cannot contain promises or errors, as these do not have useful
- * distributed equality semantics. Keys also cannot contain any CopyTagged
- * except for those recognized as CopySets, CopyBags, and CopyMaps.
- *
- * Be aware that we may recognize more CopyTaggeds over time, including
- * CopyTaggeds recognized as Keys.
- *
- * Distributed equality is location independent.
- * The same two Keys, passed to another location, will be `keyEQ` there iff
- * they are `keyEQ` here. (`keyEQ` tests equality according to the
- * key distributed equality semantics.)
+ * TODO placeholder. Figure out how to import from `@endo/patterns` instead
```

Comment:
@turadg , Since I have other successful type imports above, I thought these would be trivial. But nothing I tried worked. How should I do this?

Ditto for all occurrences of this TODO placeholder comment.

Reply:
I'll have to investigate. It might be the same hurdle I hit with bringing https://github.com/Agoric/agoric-sdk/pull/7375 into Endo.

Ok to merge with this TODO imo